setup.py

```diff
@@ -4,7 +4,7 @@
 requirements = [
     'jupyter',
     'numpy',
-    'matplotlib==3.4',
+    'matplotlib',
```

Comment:
Can you also include `matplotlib_inline` here?

Reply:
Yes, that is very much expected. We don't need to pin `matplotlib_inline` as i mentioned offline yesterday.

This [issue](https://github.com/ipython/matplotlib-inline/issues/4) will explain why we don't need to explicitly add `matplotlib-inline` as a dependency.

> IPython has a direct dependency on the matplotlib-inline backend so that the inline backend is always useable inside IPython.